server/__init__.py

```diff
@@ -352,6 +352,8 @@ def log_unhandled(e):
   app.jinja_env.globals['GA_ACCOUNT'] = app.config['GA_ACCOUNT']
   app.jinja_env.globals['NAME'] = app.config['NAME']
   app.jinja_env.globals['BASE_HTML'] = app.config['BASE_HTML_PATH']
+  app.jinja_env.globals['LOGO_PATH'] = app.config['LOGO_PATH']
+  app.jinja_env.globals['OVERRIDE_CSS_PATH'] = app.config['OVERRIDE_CSS_PATH']
```

Comment:
Can this work for feeding america? Or they worth a separate PR?

Reply:
feeding america required lots of custom updates to the header and footer, so not sure if it would work so well there. i'll add a comment to clarify